easybuild/easyconfigs/c/cairo/cairo-1.14.10-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'cairo'
+version = '1.14.10'
```

Comment:
@boegel thanks for the explanation. Gradually some of the inner `EB` mysteries are resolved...